module2/homework/transformContainers/test.cpp

```diff
@@ -1,6 +1,9 @@
 #include "gtest/gtest.h"
 
 // TODO: add proper includes
+#include <algorithm>
```

Comment:
algorithm is in the header. No point in adding it here,

Reply:
+1